zh_CN/access-control/authn/postgresql.md

```diff
@@ -29,21 +29,12 @@ CREATE TABLE mqtt_user (
 ```
 
 :::tip
-上面的示例创建了一个隐式的 `UNIQUE` 索引，当系统中有大量用户时，请确保查询使用的表已优化并使用有效的索引，以提升大量连接时的数据查找速度并降低 EMQX 负载。
+上面的示例创建了一个有助于查询的隐式 `UNIQUE` 索引字段（ `username` ）。当系统中有大量用户时，请确保查询使用的表已优化并使用有效的索引，以提升大量连接时的数据查找速度并降低 EMQX 负载。
 :::
 
 在此表中使用 `username` 作为查找条件。
 
-例如我们希望添加一位名为 `emqx_u`、密码为 `public`、盐值为 `slat_foo123`、散列方式为 `sha256` 且超级用户标志为 `true` 的用户：
-
-> PostgreSQL 中使用加密函数需要启用 pgcrypto 扩展。
-
-```sql
-postgres=# create extension pgcrypto;
-CREATE EXTENSION
-```
-
-SQL 如下：
+例如我们希望添加一位名为 `emqx_u`、密码为 `public`、盐值为 `slat_foo123`、散列方式为 `sha256` 且超级用户标志为 `true` 的用户，SQL 如下：
 
 ```sql
 INSERT INTO mqtt_user(username, password_hash, salt, is_superuser) VALUES ('emqx_u', encode(digest('public' || 'slat_foo123', 'sha256'), 'hex'), 'slat_foo123', true);
```

Comment:
we are removed the extension enable guide, so user need generate hashed password by themself.
```suggestion
INSERT INTO mqtt_user(username, password_hash, salt, is_superuser) VALUES ('emqx_u', '44edc2d57cde8d79c98145003e105b90a14f1460b79186ea9cfe83942fc5abb5', 'slat_foo123', true);
```